pkg/simple/client/auditing/elasticsearch/elasticsearch.go

```diff
@@ -322,6 +322,23 @@ func parseToQueryPart(f *auditing.Filter) interface{} {
 				},
 			})
 		}
+
+		for k, v := range f.ObjectRefWorkspaceMap {
+			bi.Should = append(bi.Should, map[string]interface{}{
+				"bool": &BoolBody{
+					Filter: []map[string]interface{}{{
+						"match_phrase": map[string]string{"Workspace.keyword": k},
+					}, {
+						"range": map[string]interface{}{
+							"RequestReceivedTimestamp": map[string]interface{}{
+								"gte": v,
+							},
+						},
+					}},
+				},
+			})
+		}
+
```

Comment:
@wanjunlei the relationship between ws  and upper ns condition is or, which may cause problems. For example, an user is a ws admin who queries on an ns and may still see the entire ws.